sdk/cosmos/azure-cosmos/test/test_retry_policy.py

```diff
@@ -241,12 +241,6 @@ def test_default_retry_policy_for_create(self):
 
             self.assertDictEqual(created_document, {})
 
-            # 3 retries for readCollection. No retry for createDocument.
-            # Counter ends up in three additional calls while doing create_item,
-            # which are reads to the database and container before creating the item.
```

Comment:
very random but what happened with this test?

Reply:
This PR actually fixes this issue and caused it to fail, the reason there were 3 additional calls were because of the old cache that was at the container level. There were a couple fo other tests that took into account the additional read calls as well.